airflow/models/connection.py

```diff
@@ -391,7 +394,14 @@ def get_connection_from_secrets(cls, conn_id: str) -> 'Connection':
         :return: connection
         """
         for secrets_backend in ensure_secrets_loaded():
-            conn = secrets_backend.get_connection(conn_id=conn_id)
-            if conn:
-                return conn
+            try:
+                conn = secrets_backend.get_connection(conn_id=conn_id)
+                if conn:
+                    return conn
+            except Exception as e:
+                log.warning('Unable to retrieve connection from alternative secret backend:'
+                            f'\n{e}\n'
+                            'Checking default secrets backends'
+                            )
```

Comment:
Do not use f-strings to format log messages. [This blog post explains it well.](https://www.siegescape.com/blog/2019/10/3/be-careful-with-your-f-strings). In this particular instance, it’s best to use `logger.exception()` to also retain the stack trace, which is useful for debugging.